src/test/java/io/appium/java_client/service/local/ThreadSafetyTest.java

```diff
@@ -86,8 +87,8 @@
                 throw isRunningTestThread2.throwable;
             }
 
-            assertTrue(isRunningTestThread.result.equals(true));
-            assertTrue(isRunningTestThread2.result.equals(true));
+            assertEquals(true, isRunningTestThread.result);
```

Comment:
cannot we just test that `assertTrue(isRunningTestThread.result)` ?

Reply:
we can't: `assertTrue` requires `boolean` input parameter, but `isRunningTestThread.result` is of `Object` type

Reply:
I'd rather use Boolean.valueOf then